app/src/main/res/layout/story_chapter_view.xml

```diff
@@ -42,6 +42,7 @@
 
         <org.oppia.android.app.customview.LessonThumbnailImageView
```

Comment:
There are files related to other configuration for this file, We need the same implementation there too. 
`layout-land/story_chapter_view.xml`
`layout-sw600dp/story_chapter_view.xml`

Reply:
Done.